expat/doc/reference.html

```diff
@@ -974,6 +974,14 @@ <h4 id="XML_ParserCreateNS">XML_ParserCreateNS</h4>
 to support RDF processors. It is a programming error to use the null separator
 with <a href= "#XML_SetReturnNSTriplet">namespace triplets</a>.</div>
 
+<p><strong>Note:</strong>
+Expat does not validate namespace URIs (beyond encoding)
+against RFC 3986 today (and is not required to so with regard to
```

Comment:
"to so" -> "to do so"

Reply:
Good catch, thanks for the review and report! :+1: Will push a fix in a second.